src/figuresClasses.ts

```diff
@@ -1,19 +1,80 @@
-export interface Figure {
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
 
+export interface Figure {
+  shape: Shape,
+  color: Color,
+  getArea(): number,
 }
 
 export class Triangle implements Figure {
+  shape: Shape;
```

Comment:
instead of writing this.shape = 'triangle' you can indicate which shape is used here :
shape: Shape = 'triangle';

do the same with 'circle' and 'rectangle'